cmake/Modules/Packages.cmake

```diff
@@ -101,8 +101,8 @@ if(TOMOPY_USE_TIMEMORY)
     find_package(TiMemory)
```

Comment:
You should probably strip out the C and C++ implementation of timemory. The package has changed a lot and I doubt this `find_package` will even work on a case-sensitive file system as I switched to just `timemory` instead of `TiMemory` a very long time ago.

Reply:
And, on that note, the python code has changed a lot too. I know for a fact that whatever is in benchmarking is broken if you download the timemory in conda-forge instead of the timemory in the jrmadsen channel

Reply:
Do you remember what version of timemory is compatible? Can I just add a version specification?

Reply:
3.0.0 was the big rewrite

Reply:
I'll resolve this since this should probably go in a different PR